exotica/include/exotica/Scene.h

```diff
@@ -336,6 +336,7 @@ namespace exotica
       void setCollisionScene(const moveit_msgs::PlanningSceneConstPtr & scene);
       CollisionScene_ptr & getCollisionScene();
       std::string getRootName();
+      std::string getRobotRootName();
```

Comment:
If you look at #114, I've changed L338 as ``Root`` is not clear and added the ability to retrieve the root joint. So in that line, perhaps rename ``getRobotRootName`` to ``getModelRootLinkName``?